pkg/controller/certificates/certificate_controller.go

```diff
@@ -73,29 +75,29 @@ func NewCertificateController(
 	csrInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			csr := obj.(*certificates.CertificateSigningRequest)
-			klog.V(4).Infof("Adding certificate request %s", csr.Name)
+			logger.V(4).Info("Adding certificate request", "csr", csr.Name)
```

Comment:
Please use `klog.KObj` for all objects of type `certificates.CertificateSigningRequest`. This applies here and below (not going to repeat it).